test/integ_tests/test_create_quantum_job.py

```diff
@@ -27,12 +27,13 @@
 from braket.jobs import Framework, get_input_data_dir, hybrid_job, retrieve_image, save_job_result
 
 
-@pytest.fixture
-def decorator_python_version(aws_session):
-    image_uri = retrieve_image(Framework.BASE, aws_session.region)
-    tag = aws_session.get_full_image_tag(image_uri)
-    major_version, minor_version = re.search(r"-py(\d)(\d+)-", tag).groups()
-    return major_version, minor_version
+# todo: capture python version from container
+# @pytest.fixture
+# def decorator_python_version(aws_session):
+#     image_uri = retrieve_image(Framework.BASE, aws_session.region)
+#     tag = aws_session.get_full_image_tag(image_uri)
+#     major_version, minor_version = re.search(r"-py(\d)(\d+)-", tag).groups()
+#     return major_version, minor_version
```

Comment:
I'll capture a follow-up to go back to getting the python version from the container in the code, but setting manually to 3.10 to unblock